src/transformers/modeling_utils.py

```diff
@@ -75,7 +77,7 @@
 )
 from .utils.hub import convert_file_size_to_int, get_checkpoint_shard_files
 from .utils.import_utils import ENV_VARS_TRUE_VALUES, is_sagemaker_mp_enabled
-from .utils.quantization_config import BitsAndBytesConfig
+from .utils.quantization_config import AutoGPTQConfig, BitsAndBytesConfig, QuantizationMethod
```

Comment:
I would call this just `GPTQConfig` (or if we can `GptqConfig`, casing hurts my eyes ;-)): the `Auto` prefix makes me think of the auto APIs ;-)